ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Protocol.hs

```diff
@@ -236,14 +283,14 @@ rewind k p =
 
 update :: forall xs. CanHardFork xs
        => ConsensusConfig (HardForkProtocol xs)
-       -> Ticked (HardForkLedgerView xs)
        -> OneEraValidateView xs
-       -> HardForkChainDepState xs
+       -> Ticked (HardForkLedgerView xs)
+       -> Ticked (HardForkChainDepState xs)
        -> Except (HardForkValidationErr xs) (HardForkChainDepState xs)
 update cfg@HardForkConsensusConfig{..}
-       (Ticked slot ledgerView)
        (OneEraValidateView view)
-       chainDepState =
+       (Ticked slot ledgerView)
+       (Ticked _slot chainDepState) =
```

Comment:
We should not be aligning here. Instead to the same as in `check`.

Reply:
Done